test/regression/convert/covert_test.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
I think this file is named wrong, should be `convert_test.go` right?

Reply:
Done :thanks: